### DISCUSSION TITLE
When I log in for the first time, the time tracking is open in the left menu, can I change it to open the 'Administration' part at the first login?

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4980] [Plugins]
![스크린샷 2024-07-19 103701](https://github.com/user-attachments/assets/f6fc9b79-f4f9-4fdf-92a4-b51a8ddab4cc)
![스크린샷 2024-07-19 103719](https://github.com/user-attachments/assets/ac0788ce-bc7e-4cfb-bf6f-e231d5e255fc)

Hello.

When I log in for the first time, the 'Time Tracking' is open in the left menu, can I change it to open the 'Administration' part at the first login?

Which part of the code should I modify?

Please reply.

**Comment by kevinpapst** [comment]
Check the wizards how they intercept the first login of a user.
You might be able to re-use this logic.

**Reply by seleondevel** [reply]
@kevinpapst Can you give me some more hints? What do you mean by checking how to wizard-intercept your first login?

**Reply by kevinpapst** [reply]
No, I can't. If you want proper consulting, get your company to pay for it. 
Otherwise there has never been a person posting so many question and taking up so much of my personal and UNPAID time.
